lib/resources/r4/related_person.yaml

```diff
@@ -107,6 +104,31 @@ fields:
             }
           ]
         }
+        {
+          "extension": [
+            {
+              "valueCodeableconcept": {
+                "coding": [
+                  {
+                    "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+                    "code": "SIS",
+                    "display": "sister",
+                    "userSelected": false
+                  }
+                ],
+                "text": "Sister"
+              },
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/relation"
```

Comment:
Should this relation extension be removed?

Reply:
removed here https://github.com/cerner/fhir.cerner.com/pull/624/commits/aa0cea8db4ceaec2e3883cc85832dcd9f20e40c2#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6L109-L122